src/controllers/admin/articles/index.js

```diff
@@ -5,7 +5,7 @@ const router = new Router();
 import {defineAdminPageRoute} from '../../helpers';
 import Article from '../../../models/article';
 import {fetchArticles} from '../../../react/actions/articles';
-
+const formidable = require('express-formidable');
```

Comment:
You could use import instead of require. This is a static import instead of a dynamic require which is faster.